OsmAnd/res/values/strings.xml

```diff
@@ -343,9 +343,9 @@
     <string name="shared_string_tracks">Tracks</string>
     <string name="routing_attr_driving_style_name">Driving style</string>
     <string name="route_altitude">Route elevation</string>
-    <string name="altitude_descent">◺</string>
-    <string name="altitude_ascent">◿</string>
-    <string name="altitude_range">◿◺◺◿</string>
+    <string name="altitude_descent">Descent</string>
```

Comment:
I seem to condescend on the notion of whether maybe the combination "◺ Descent"makes it easier to understand?